simmechanics/data/icub2_5/ICUB_2-5_BB_simmechanics_options.yaml.in

```diff
@@ -8,7 +8,10 @@ originRPY: [0.0,0.0,3.14]
 # Meshes options
 scale: "0.001 0.001 0.001"
 forcelowercase: Yes
-filenameformatchangeext: "package://iCub/meshes/simmechanics/%s-binary.stl"
+@MESH_FOLDER@
```

Comment:
I would actually call this `@MESH_FILE_FORMAT@` instead.

Reply:
Yes, better!